crops/productionfunctions.go

```diff
@@ -45,9 +51,10 @@ func isLeapYear(year int) bool {
 func (p productionFunction) GetCumulativeMonthlyProductionCostsEarly() []float64 {
 	return p.cumulativeMonthlyProductionCostsEarly
 }
-func cumulateMonthlyCosts(mc []float64, fc []float64, start time.Time, daysToMaturity int) ([]float64, float64) {
+func cumulateMonthlyCosts(mc []float64, fc []float64, start time.Time, daysToMaturity int) ([]float64, float64, float64) {
```

Comment:
look at you! great work adding the additional return value. the only concern is that it seems like it might only get used once, so maybe the pattern here is not really well developed. i like what you are thinking, and how you are applying this new language though!